db/migrate/20200810121128_remove_subscriber_count_from_digest_run.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSubscriberCountFromDigestRun < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :digest_runs, :subscriber_count # rubocop:disable Rails/ReversibleMigration
```

Comment:
no biggie, but it'd have been less characters to add `, :integer` here than to ignore the rule.

Reply:
I see what you're saying. While that would make the column replacement, that's not true for the data it holds i.e. this is also a data migration to delete everything. It's in that sense that I'm saying it's irreversible (there's not much point in reversing it).

Reply:
Would the linting error go away if this was in a `def up` block rather than a `def change` block (and then there wasn't a `def down` block).

Reply:
So it does - updated, thanks for the suggestion.